php/WP_CLI/CommandWithTranslation.php

```diff
@@ -125,6 +140,112 @@ public function install( $args, $assoc_args ) {
 	}
 
 	/**
+	 * Updates the active translation of core, plugins, and themes.
+	 *
+	 * [--dry-run]
+	 * : Preview which translations would be updated.
+	 *
+	 * [--force]
+	 * : Force the check for updates.
```

Comment:
We should make `--force` the default behavior, and drop the argument. `wp (plugin|theme) list` both force refresh updates, which I think is an acceptable behavior.